public/app/features/provisioning/Shared/RepositoryList.tsx

```diff
@@ -39,36 +45,42 @@ export function RepositoryList({ items }: Props) {
 
     if (filteredItems.length) {
       return (
-        <Stack>
-          <Alert title={''} severity="info">
-            <Trans
-              i18nKey="provisioning.folder-repository-list.partial-managed"
-              values={{ managedCount, resourceCount }}
-            >
-              {{ managedCount }}/{{ resourceCount }} resources managed by Git sync.
-            </Trans>
-            {unmanagedCount > 0 && (
-              <>
-                {' '}
-                <Trans i18nKey="provisioning.folder-repository-list.unmanaged-resources" count={unmanagedCount}>
-                  {{ count: unmanagedCount }} resources aren&apos;t managed by Git sync.
-                </Trans>
-              </>
-            )}
-            {isFreeTierLicense() && (
-              <>
-                <br />
-                <Trans i18nKey="provisioning.free-tier-limit.message-connection">
-                  Free-tier accounts are limited to 20 resources per folder. To add more resources per folder,
-                </Trans>{' '}
-                <TextLink href={UPGRADE_URL} external>
-                  <Trans i18nKey="provisioning.free-tier-limit.upgrade-link">upgrade your account</Trans>{' '}
-                </TextLink>
-                .
-              </>
-            )}
-          </Alert>
-        </Stack>
+        <Alert
+          title={''}
+          severity="info"
+          buttonContent={
+            unmanagedCount > 0 ? (
+              <Trans i18nKey="provisioning.folder-repository-list.export-remaining-resources-button">
+                Export remaining resources
+              </Trans>
+            ) : undefined
+          }
+          onRemove={unmanagedCount > 0 ? handlePushUnmanaged : undefined}
```

Comment:
Not sure how useful this is - it redirects to the dashboards page where it's not entirely clear what needs to be done to start exporting.